modules/ROOT/partials/misc/inline-formatting-of-list-bullets.adoc

```diff
@@ -0,0 +1,26 @@
+== Inline formatting.
+
+List bullets and list numbers can new be inline-formatted.
+
+Inline-formatting is the direct formatting of an object. It contrasts with formatting applied to an object because that object is a particular type of object.
+
+If all Table Headers are formatted a particular way, that is not inline formatting. When a particular Table Header is selected and has formatting specifically applied to it, that is inline formatting.
+
+Inline-formatting is also known as spot-formatting.
+
+If an end-user selects the entire content of a list item and applies inline formatting, such as a change of color or change of font-size, this spot-formatting is also applied to the list item’s associated bullet or number.
+
+Only the bullets or list numbers associated with the selection take on the inline-formatting applied to the selection.
```

Comment:
Can this be implied in the above paragraph instead?

Reply:
> Can this be implied in the above paragraph instead?

It is more than implied, it is outright stated in the above paragraph. To wit:

*The above paragraph (with the outright statement emphasised)*  
> If an end-user selects the entire contents of a list item and
> applies inline formatting — such as a change of color or change of
> font-size — this spot-formatting is also applied **to the list
> item’s associated bullet or number**.

The following paragraph, in effect, repeats this but emphasises that **only** the associated list livery takes on the applied formatting.

There’s an old journalist’s maxim pertaining to getting a message across (that is, delivering information such that said information is actually taken in by the recipient):

1. Tell ’em.

2. Tell ’em you told ’em.

3. Tell ’em again.

It’s a truism because journalists, in common with technical writers, need to be aware their readers are, for the most part, only skimming their words.

So repetition is key to getting information across.

My two paragraphs here don’t quite follow the old maxim, because I’ve left out the second step, *tell ’em you told ’em*, but the repetition-with-variation is in step with the maxim, nonetheless.